mj_envs/utils/quat_math.py

```diff
@@ -44,6 +44,18 @@ def axis_angle2quat(axis, angle):
     s = np.sin(angle/2)
     return np.array([c, s*axis[0], s*axis[1], s*axis[2]])
 
+def quatMagnitude(quat):
```

Comment:
Have you looked at this function. When dt =1, it will return you this magnitude
```
def quat2Vel(quat, dt=1):
    axis = quat[1:].copy()
    sin_a_2 = np.sqrt(np.sum(axis**2))
    axis = axis/(sin_a_2+1e-8)
    speed = 2*np.arctan2(sin_a_2, quat[0])/dt
    return speed, axis
 ```